web/src/app/admin/skills/skills.component.css

```diff
@@ -0,0 +1,8 @@
+ .btn{
```

Comment:
remove css files related to component, later while design it will be consider

Reply:
Done